src/ralph/admin/autocomplete.py

```diff
@@ -189,6 +201,25 @@ def get_base_ids(self, model, value):
         queryset = self.get_query_filters(queryset, value, search_fields)
         return queryset[:self.limit].values_list('pk', flat=True)
 
+    def get_results(self, user, can_edit):
+        results = [
```

Comment:
this list comprehension could be at least separated to another function (this is the same code as above).